ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -934,6 +934,19 @@ public MessageInfo findKey(StoreKey key) throws StoreException {
     }
   }
 
+  @Override
+  public Map<StoreKey, MessageInfo> findKeys(List<? extends StoreKey> storeKeys) {
+    Map<StoreKey, MessageInfo> messageInfoMap = new HashMap<>();
```

Comment:
TODO: Need to look into the possibility if its more efficient to not call `findKey()` multiple times in a loop, and instead search the index once for all the blobs in the list. Will do so in a separate ticket. For now,  calling `findKey()` multiple times in a loop is no worse than what was happening before for the case of `BlobStore`.